TON DNS materialized view (based on #8065)

shuva10v opened the pull request:
## Thank you for contributing to Spellbook 🪄
Please open the PR in **draft** and mark as ready when you want to request a review. 

### Description:

Based on #8065


---
quick links for more information:
- [README.md](https://github.com/duneanalytics/spellbook/blob/main/README.md)
- [spellbook docs](https://github.com/duneanalytics/spellbook/tree/main/docs)
- [CONTRIBUTING.md](https://github.com/duneanalytics/spellbook/blob/main/CONTRIBUTING.md)

jeff-dude reviewed (commented):
even though the schema differs, do you want to move this into
dbt_subprojects/daily_spellbook/models/ton
so all models are grouped together?
you could create sub directory in there, if you wanted to slightly separate

dune-eng commented:
Workflow run id 15024149731 approved.

dune-eng commented:
Workflow run id 15024149875 approved.

dune-eng commented:
Workflow run id 15025507447 approved.

dune-eng commented:
Workflow run id 15025507467 approved.

dune-eng commented:
Workflow run id 15025507732 approved.

dune-eng commented:
Workflow run id 15025896996 approved.

shuva10v commented:
> even though the schema differs, do you want to move this into dbt_subprojects/daily_spellbook/models/ton so all models are grouped together? you could create sub directory in there, if you wanted to slightly separate

moved to __ton__ subfolder, also added one additional view

dune-eng commented:
Workflow run id 15037039969 approved.

dune-eng commented:
Workflow run id 15037039964 approved.

dune-eng commented:
Workflow run id 15037040320 approved.
